dts/bindings/pinctrl/sifive,pinctrl.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2022 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  SiFive FE310 IO Function (iof)
+  Binding covers the IOF_EN/IOF_SEL registers that are a subset of the GPIO
+  controller.
```

Comment:
nitpick: please extend this documentation on how pinctrl is expected to be used (see other vendors as a reference)

Reply:
Done.